base/libgit2/consts.jl

```diff
@@ -351,6 +351,13 @@ These are used to select which global option to set or get and are used in `git_
                     SET_SSL_CERT_LOCATIONS   = 12)
 
 
+    """
+Option flags for `GitProxy`.
```

Comment:
could fix it if you or someone else is feeling charitable, doesn't have to be in this pr tho